docs/modules/nexus/pages/index.adoc

```diff
@@ -1,2 +1,40 @@
-= Nexus
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+= Nexus 3
+
+Nexus 3 is an open source repository management platform developed by Sonatype to host and provide different registry and repository formats to store software artifacts.
+
+This repository contains the logic and configuration for a central Nexus 3 repository instance to store build artifacts, provided by Jenkins jobs or provide access to shared and proxied repositories.
+
+== Setup
+
+The OpenShift templates are located in `ocp-config` and can be compared with the OC cluster using https://github.com/opendevstack/tailor[tailor]. For example, run `cd ocp-config && tailor status` to see if there is any drift between current and desired state.
+
+You are also able to use the script `prepare-nexus.sh` located in the `infrastructure-setup/scripts` folder of this repository.
+You will have to customize the script to match your target environment.
+
+If you have a running Nexus 3 instance and you only want to add scripted resources, take a look at the `create-nexus-resources.sh` script. +
+Inside this script you will find curl calls, that are used to run scripts in the Nexus 3 scripting API. +
+The called scripts are located in the `json` subfolder and can be customized by your need. +
+*Please keep in mind, that the user, who is provided in the curl calls has to have script execution rights in Nexus 3!*
+
+The `createProjectSpecificAccess.json` contains the logic to create a project specific nexus user to isolate project artifacts from each other. +
+The logic uses content selectors with an expression that uses the `path` to identify repos, that belong to one project, so you will have to specify a naming convention, if you use private repositories and isolated projects.
```

Comment:
I'm not sure I am getting this. Is the CSEL now a path within a repo, or a repo? I thought the former, but this paragraph makes me unsure now.

Reply:
CSEL is the expression language used for a content selector and you are right, it is used to retrieve artifacts from an existing repository matching the expression. In the provided implementation the content selector is set to match all repositories, so even older and new repositories will match.